src/ButtonThread.cpp

```diff
@@ -180,32 +189,27 @@ int32_t ButtonThread::runOnce()
             // If a nag notification is running, stop it and prevent other actions
             if (moduleConfig.external_notification.enabled && (externalNotificationModule->nagCycleCutoff != UINT32_MAX)) {
                 externalNotificationModule->stopNow();
-                return 50;
-            }
-#ifdef BUTTON_PIN
-#if !defined(USERPREFS_BUTTON_PIN)
-            if (((config.device.button_gpio ? config.device.button_gpio : BUTTON_PIN) !=
-#endif
-#if defined(USERPREFS_BUTTON_PIN)
-            if (((config.device.button_gpio ? config.device.button_gpio : USERPREFS_BUTTON_PIN) !=
-#endif
-                 moduleConfig.canned_message.inputbroker_pin_press) ||
-                !(moduleConfig.canned_message.updown1_enabled || moduleConfig.canned_message.rotary1_enabled) ||
-                !moduleConfig.canned_message.enabled) {
-                powerFSM.trigger(EVENT_PRESS);
-            }
-#endif
-#if defined(ARCH_PORTDUINO)
-            if ((settingsMap.count(user) != 0 && settingsMap[user] != RADIOLIB_NC) &&
-                    (settingsMap[user] != moduleConfig.canned_message.inputbroker_pin_press) ||
-                !moduleConfig.canned_message.enabled) {
-                powerFSM.trigger(EVENT_PRESS);
+                break;
             }
+#ifdef ELECROW_ThinkNode_M1
+            sendAdHocPosition();
```

Comment:
because we had concerns - the button is not easy to press, much unlike on T-Echo